mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/service/impl/IdentityServiceImpl.java

```diff
@@ -54,4 +65,36 @@ public IdentityData getIdentity(String individualId) throws MockIdentityExceptio
 		return identityData;
 	}
 
+	@Override
+	public void addVerifiedClaim(VerifiedClaimRequestDto verifiedClaimRequestDto) throws MockIdentityException {
+		VerifiedClaim verifiedClaim =null;
+		//validateVerifiedClaim(verifiedClaimRequestDto);
+		String idHash= HelperUtil.generateB64EncodedHash(ALGO_SHA3_256, verifiedClaimRequestDto.getIndividualId()+ verifiedClaimRequestDto.getTrustFramework()+ verifiedClaimRequestDto.getClaim());
```

Comment:
Are we not checking if the individualId is present in the identity table? And let's use the lowercase of trust framework to generate the hash.

Reply:
added individualId check